go/ct/common/bytes_test.go

```diff
@@ -99,3 +101,42 @@ func TestBytes_InvalidJsonFails(t *testing.T) {
 		})
 	}
 }
+
+func TestBytes_RandomBytesProducesRandom(t *testing.T) {
+	rnd := rand.New()
+	for i := 0; i < 10000; i++ {
+		b1 := RandomBytes(rnd, 100)
+		allZeros := true
+		for _, b := range b1.data {
+			if b != 0 {
+				allZeros = false
+				break
+			}
+		}
+
+		if len(b1.data) > 1 && allZeros {
+			t.Errorf("RandomBytes produced all zeros: %v, len:v %v", b1.data, len(b1.data))
+		}
```

Comment:
There is a certain probability that randomly something like `[]byte{0,0}` is generated, and it would be valid.
Also, this test would pass if `RandomBytes` would always return `[]byte{0,1}`.

Suggestion for an alternative testing strategy: generate 5 random byte sequences and check that there are no collisions.